src/components/datePicker/DatePickerBox.tsx

```diff
@@ -14,6 +14,7 @@ export interface IDatesSelectionBox {
   title: string;
   quickOptions?: IOption[];
   isRange?: boolean;
+  rangeLimit?: IRangeLimit;
```

Comment:
I'm wondering if you could do something with the `selectionRules` prop a few lines down instead. Sometimes, validation is more complex than a simple range limit and it kind of duplicates the rules.